### DISCUSSION TITLE
Предложения по проекту

### DISCUSSION THREAD

**Discussion by DaovaJo** [DaovaJo/Isys_engineering#2] [Ideas]
Привет, если у вас есть предложения по проекту, чтобы мы потом часами не искали в общей беседе, что там за идея классная у кого-то была, пишите сюда, здесь будет легче найти и сразу без воды обсудить ее

**Comment by MariaDanu** [comment]
Я пока предлагаю, чтобы у нас был свой чат с нашим словарем, Можно пока без сохранения сообщений (архив), просто чат с одним человеком, для проверки работы нашего словаря. Если идея у нас зайдет, то можно его продлить на  диплом как расширение функционала к любому мессенджеру

**Comment by vikulkin** [comment]
я тоже думаю,что пока нужно сделать для 1 человека чат и посмотреть ,что и как,что можно добавить,главное не перегрузить его

**Comment by Sertyjop** [comment]
Одностраничное подобие сайта авторизация через никнейм(в будущем создание приложения на подобие telegram). 
Графика-Figma

**Comment by Bibitriy** [comment]
DRAW IO

**Comment by Bibitriy** [comment]
[Концептуальная модель.docx](https://github.com/DaovaJo/Isys_engineering/files/9778128/default.docx)

**Comment by Sertyjop** [comment]
Полина
загрузила 2 диаграммы, use case - ее приняли и DFD - ее надо будет редактировать, на данный момент она сойдет, в файле DFD2.0, старая версия,  в файле DFD2.1 та же диаграмма, только с возможными изменениями
